### DISCUSSION TITLE
PiPedal v1.3.59 Release

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#231] [General]
## PiPedal 1.3.59 Release

20% performance improvement when using Toob Neural Amp Modeller with large Wavenet models.

Minor bug fixes:
- Display file extensions in File Property dialogs.

(re-issue of v1.3.58)

<hr /><em>This discussion was created from the release <a href='https://github.com/rerdavies/pipedal/releases/tag/v1.3.59'>PiPedal v1.3.59 Release</a>.</em>

**Comment by 38github** [comment]
Toob Neural Amp Modeler doesn't beep now on my setup. Thank you!
